balancer/weightedroundrobin/weightedroundrobin.go

```diff
@@ -44,23 +49,13 @@ func (a AddrInfo) Equal(o interface{}) bool {
 
 // SetAddrInfo returns a copy of addr in which the BalancerAttributes field is
 // updated with addrInfo.
-//
-// # Experimental
-//
-// Notice: This API is EXPERIMENTAL and may be changed or removed in a
-// later release.
 func SetAddrInfo(addr resolver.Address, addrInfo AddrInfo) resolver.Address {
 	addr.BalancerAttributes = addr.BalancerAttributes.WithValue(attributeKey{}, addrInfo)
 	return addr
 }
 
 // GetAddrInfo returns the AddrInfo stored in the BalancerAttributes field of
 // addr.
-//
-// # Experimental
-//
-// Notice: This API is EXPERIMENTAL and may be changed or removed in a
-// later release.
 func GetAddrInfo(addr resolver.Address) AddrInfo {
 	v := addr.BalancerAttributes.Value(attributeKey{})
 	ai, _ := v.(AddrInfo)
```

Comment:
Do we want to move these helpers to another package now? (Doesn't have to be part of this PR). These helpers have nothing to do with the balancer implementation present.

Reply:
Yes!  I was hoping you'd do that in your other PR that uses them.  But we can move them any time.

Reply:
Ok. I'll make an issue to move these helpers, maybe I can take a look this week since I'm on call.

Reply:
Made https://github.com/grpc/grpc-go/issues/6261